source/renderer/zettlr-editor.js

```diff
@@ -531,9 +542,9 @@ class ZettlrEditor {
     // We use 50 characters in order to get an approximately correct width (clientWidth is an integer)
     var container = document.createElement('div')
     container.id = 'measureMonoWidth'
-    container.innerHTML = '<span>--------------------------------------------------</span>'
+    container.innerHTML = '<span>' + '-'.repeat(100) + '</span>'
     this._cm.getWrapperElement().appendChild(container)
-    this._monospaceWidth = (container.clientWidth + 1) / 50
+    this._monospaceWidth = (container.clientWidth + 1) / 100
```

Comment:
If I understand your approach correct, you're trying to guess the character width by seeing how far a container can go — what does this do on 4k monitors? Is this not a problem, or does it work regardless of screen size? If the former one, we should try to implement as many edge-cases as possible to not have to deal with this in the future again!

Reply:
The character size is measured by creating a span with a lot of spaces in it, and then take the width of this span. This should work regardless of the system. The applied css prevents that this span is ever displayed, so there shouldn't be any side effects.